A3-channel-tracing.md

```diff
@@ -86,25 +73,15 @@ message ChannelTraceEvent {
   }
 }
 
-message ChannelTraceData {
+message ChannelTrace {
   // Number of events ever logged in this tracing object. This can differ from
   // events.size() because events can be overwritten or garbage collected by
   // implementations.
-  int64 num_events_logged = 3;
+  int64 num_events_logged = 1;
   // Time that this channel was created.
-  google.protobuf.Timestamp channel_created_timestamp = 4;
+  google.protobuf.Timestamp creation_time = 2;
   // List of events that have occurred on this channel.
-  repeated ChannelTraceEvent events = 5;
-}
-
-message ChannelTrace {
-  // Ref to the channel or subchannel this trace refers to.
-  oneof ref {
-    ChannelRef channel_ref = 1;
-    SubchannelRef subchannel_ref = 2;
-  }
-  // All of the data for this channel.
-  ChannelTraceData channel_data = 3;
+  repeated ChannelTraceEvent events = 3;
   // Optional, only present if this channel has children
   repeated ChannelTrace child_data = 4;
```

Comment:
Why is this field needed?  In the channelz API, every node in the tree has its own `ChannelTrace`, so why do we need the `ChannelTrace` for an individual node to directly contain the `ChannelTrace` for its child nodes?

Reply:
Done.